tests/acceptance/acceptancesuite.js

```diff
@@ -160,6 +173,213 @@ test(`Facets load on the page, and can affect the search`, async t => {
   await t.expect(actualResultsCount).eql(initialResultsCount);
 });
 
+test(`radius filter filterbox works with back/forward navigation and page refresh`, async t => {
+  const radiusFilterLogger = RequestLogger({
+    url: /v2\/accounts\/me\/answers\/vertical\/query/
+  });
+  await t.addRequestHooks(radiusFilterLogger);
+  await registerIE11NoCacheHook(t);
+  const searchComponent = FacetsPage.getSearchComponent();
+  await searchComponent.enterQuery('all');
+  await searchComponent.submitQuery();
+
+  const filterBox = FacetsPage.getStaticFilterBox();
+  const radiusFilter = await filterBox.getFilterOptions('DISTANCE');
+
+  // Choose the 25 miles radius filter
+  await radiusFilter.toggleOption('25 miles');
+  const filterTags = Selector('.yxt-ResultsHeader-removableFilterTag');
+  await t.expect(filterTags.count).eql(1);
+  await expectRequestLocationRadiusToEql(radiusFilterLogger, 40233.6);
+  radiusFilterLogger.clear();
+
+  // Hit the back button
+  await browserBackButton();
+  await t.expect(filterTags.count).eql(0);
+  await expectRequestDoesNotContainParam(radiusFilterLogger, 'locationRadius');
+  radiusFilterLogger.clear();
+
+  // Hit the forward button
+  await browserForwardButton();
+  await t.expect(filterTags.count).eql(1);
+  await expectRequestLocationRadiusToEql(radiusFilterLogger, 40233.6);
+  radiusFilterLogger.clear();
+
+  // Refresh the page
+  await browserRefreshPage();
+  await t.expect(filterTags.count).eql(1);
+  await expectRequestLocationRadiusToEql(radiusFilterLogger, 40233.6);
+});
+
+test(`static filter filterbox works with back/forward navigation and page refresh`, async t => {
```

Comment:
Does the SDK treat location radius different than other static filters? If not, I think we'd be fine with just this test here.

Reply:
yup it does! locationRadius is a different query param than filters. That being said FilterBox probably doesn't treat them different enough to warrant an additional acceptance test. Originally I added this test because I had issues with backnav with only a RADIUS_FILTER filterbox, but that was due to one particular issue I think I can add a jest test for.

Reply:
Actually, is it alright if I just rephrase it into one test for singleoption and one test for multioption? That's probably a more
important distinction, especially since there's two separate pathways within the component for seeding the component state depending on that. RADIUS_FILTER vs STATIC_FILTER just change the filternode that is created, and which core method we call to store the filternode.